Sources/AblyAssetTrackingCore/Model/ClientType.swift

```diff
@@ -0,0 +1,6 @@
+
+
+public enum ClientType: String, Codable {
+    case publisher = "PUBLISHER"
+    case subscriber = "SUBSCRIBER"
```

Comment:
until this is for internal use, it's not a good practice to assign a string name to string type enum, especially uppercased. You can leave it as it is if an uppercased name is required by an external API
```suggestion
    case publisher 
    case subscriber
```

Reply:
That's an existing piece of code I didn't actually write, unfortunately git is confused. I just moved the file into Core module, because it is being shared between 2 other modules (Publisher and Subscriber). I found it odd as well, so I asked Kacper why this is done, and he said if it were done again, it would not be there.